includes/wc-template-functions.php

```diff
@@ -2456,7 +2456,7 @@ function wc_display_item_meta( $item, $args = array() ) {
 		) );
 
 		foreach ( $item->get_formatted_meta_data() as $meta_id => $meta ) {
-			$value = $args['autop'] ? wp_kses_post( $meta->display_value ) : wp_kses_post( make_clickable( trim( strip_tags( $meta->display_value ) ) ) );
+			$value = $args['autop'] ? wp_kses_post( $meta->display_value ) : wp_kses_post( make_clickable( trim( $meta->display_value ) ) );
```

Comment:
If remove `strip_tags()` we can remove `make_clickable()` too.

`get_formatted_meta_data()` source code: https://github.com/woocommerce/woocommerce/blob/c92d6335ffb327825187e94fd9ecca95095fe05d/includes/class-wc-order-item.php#L210